File: nomad-samples/js/video-compare/video-compare/get-channel-frame-manifest.js
export default async function getChannelFrameManifest(AUTH_TOKEN, VIDEO_TRACKING_ID, FRAME_DT) 
{
    // Create header for the request
    const HEADERS = new Headers();
    HEADERS.append("Content-Type", "application/json");
    HEADERS.append("Authorization", `Bearer ${AUTH_TOKEN}`);
  
    const BODY = {
        VideoTrackingId: VIDEO_TRACKING_ID,   
        FrameDt: FRAME_DT
    };

    // Post
    const RESPONSE = await fetch(`https://${videoCompareApiUrl}/api/videocompare/tracking/manifest`, {
        method: "GET",
        headers: HEADERS,
        body: JSON.stringify(BODY)
    }).catch((exception) => {
        throw exception;
    });

    // Check for success
    if (RESPONSE.ok) {
        // Get the JSON from the response
        const INFO = await RESPONSE.json();

        return INFO;
    }
		
  	// There was an error
    await apiExceptionHandler(RESPONSE, "Getting channel frame mainifest failed");
}

File: nomad-samples/video-compare/py/video_compare_api/video_compare/get_channel_frame_manifest.py
from constants.project_constants import *
from exceptions.api_exception_handler import *

import json, requests

def get_channel_frame_manifest(AUTH_TOKEN: str) -> dict:

    # Check for valid parameters
    if (not AUTH_TOKEN):
        raise Exception("Authentication Token: The authentication token is invalid")

    API_URL = VIDEO_COMPARE_API_URL + "/videocompare/tracking/manifest"
        
    # Create header for the request
    HEADERS = {
  	    "Authorization": "Bearer abc...xyz",
		    "Content-Type": "application/json"
    }

    # Build the payload BODY
    BODY = {
        "VideoTrackingId": "abc00000-0000-0000-0000-000000000000",   
        "FrameDt": "2022-10-05T17:22:07+05:00"
    }

    try:
        # Send the request
        RESPONSE = requests.post(API_URL, headers= HEADERS, data= json.dumps(BODY))

        # Get the response
        

        if not RESPONSE.ok:
            raise Exception()

        

        return json.loads(RESPONSE.text)

    except:
        raise Exception("Search failed")

